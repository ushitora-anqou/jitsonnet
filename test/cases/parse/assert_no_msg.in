assert x == 3.0; 0.0
