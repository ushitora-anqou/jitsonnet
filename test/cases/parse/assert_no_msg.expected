{ Syntax.v =
  (Syntax.Assert
     ({ Syntax.v =
        ({ Syntax.v =
           (Syntax.Binary
              ({ Syntax.v = (Syntax.Var "x");
                 loc =
                 { Syntax.fname = "test/cases/parse/assert_no_msg.in";
                   ran =
                   (Some { Syntax.startpos = { Syntax.line = 1; column = 7 };
                           endpos = { Syntax.line = 1; column = 8 } })
                   }
                 },
               `Equal,
               { Syntax.v = (Syntax.Number 3.);
                 loc =
                 { Syntax.fname = "test/cases/parse/assert_no_msg.in";
                   ran =
                   (Some { Syntax.startpos = { Syntax.line = 1; column = 12 };
                           endpos = { Syntax.line = 1; column = 15 } })
                   }
                 }));
           loc =
           { Syntax.fname = "test/cases/parse/assert_no_msg.in";
             ran =
             (Some { Syntax.startpos = { Syntax.line = 1; column = 7 };
                     endpos = { Syntax.line = 1; column = 15 } })
             }
           },
         None);
        loc =
        { Syntax.fname = "test/cases/parse/assert_no_msg.in";
          ran =
          (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
                  endpos = { Syntax.line = 1; column = 15 } })
          }
        },
      { Syntax.v = (Syntax.Number 0.);
        loc =
        { Syntax.fname = "test/cases/parse/assert_no_msg.in";
          ran =
          (Some { Syntax.startpos = { Syntax.line = 1; column = 17 };
                  endpos = { Syntax.line = 1; column = 20 } })
          }
        }));
  loc =
  { Syntax.fname = "test/cases/parse/assert_no_msg.in";
    ran =
    (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
            endpos = { Syntax.line = 1; column = 20 } })
    }
  }